apps/pigment-css-next-app/src/app/page.tsx

```diff
@@ -112,6 +113,11 @@ export default function Home() {
           },
         }}
       >
+        <Stack spacing={{ xs: 2, md: 3 }}>
+          <div>Item1</div>
+          <div>Item1</div>
+          <div>Item1</div>
+        </Stack>
```

Comment:
Here is the result:  

<img width="510" alt="image" src="https://github.com/mui/pigment-css/assets/18292247/cff91112-c3e6-49f0-803b-8e52e24c800a">